_directory/data/walderwerber/data.yml

```diff
@@ -0,0 +1,29 @@
+---
+# name (required)
+# may be your real name or your nick
+name: walder,werber werbeagentur ag
+
+# geo location (required)
+# provide your favored level of detail by number of digits (like 50.107811 vs 50.11)
+# you may use services like openstreetmap.org to find out your geo location
+latitude: 47.3501404
+longitude: 8.7196706
+
+# bio
+# describe yourself, your involvement in the project and the community or whatever is relevant for you.
+# should be 300 chars at maximum. no markup or line breaks!
+bio: "Hey guys! We're a team of 3 working at walder,werber, a small marketing agency located at uster, switzerland."
+
+# image or gravatar
+# provide image file from current folder (like 'johndoe.jpg') or gravatar url (generated via https://3v4l.org/OXG7H)
+# should be square and about 200-300 px size!
+#image: johndoe.jpg
```

Comment:
I guess this needs to be left blank instead of commented out.

Reply:
Technically, it both works.